ports/servoshell/egl/ohos.rs

```diff
@@ -220,50 +253,60 @@ extern "C" fn on_surface_created_cb(xcomponent: *mut OH_NativeXComponent, window
     let xc_wrapper = XComponentWrapper(xcomponent);
     let window_wrapper = WindowWrapper(window);
 
-    // Todo: Perhaps it would be better to move this thread into the vsync signal thread.
-    // This would allow us to save one thread and the IPC for the vsync signal.
-    //
-    // Each thread will send its id via the channel
-    let _main_surface_thread = thread::spawn(move || {
-        let (tx, rx): (Sender<ServoAction>, Receiver<ServoAction>) = mpsc::channel();
+    if !SERVO_CHANNEL.get().is_some() {
```

Comment:
This looks racy to me. You could use one of the `Once` or `Lazy` types

Reply:
I don't quite understand, the SERVO_CHANNEL is a OnceLock already.